### ISSUE TITLE
apt install task-mate-desktop can not correctly install whole Mate desktop

### ISSUE THREAD

**Issue by JanuszChmiel** [sp4rkie/debian-on-termux#59]
Sure, I would try to use package diverd with DPKG to solve problematic packages such as UUUdisk. But I AM wondering, why those packages are incompatible with Proot environment. Sure, mate work even if dpkg did not correctly finished its installation including mate-core.
Could you look at this problem please?

**Comment by sp4rkie** [comment]
could you please give more details what exactly does not work for you? What commands fail? What are the results you expect but are missing etc...

**Comment by JanuszChmiel** [comment]
It is very probably The problem caused by The debian ARM64 packagers. apt
install mate and apt install task-mate-desktop can not successfully complete
Mate desktop installation.
Some package with Udisks fails and this cause The error, that installation
of Mate finish with errors.
I will install yours Debian script again to bring The exact error log.

**Comment by sp4rkie** [comment]
there is not enough information available about what the problem is. so I will close this for now.

**Comment by JanuszChmiel** [comment]
I AM very sorry. i have lost my motivation, because I have started to use
ArchLinux installed by script from Rausty. advantages are.
1. Exit command correctly closes shlee even if AN X11 session is active.
2. Script from Rausty support work around which allow to overcome The
Android 8 and newer security restriction related to /proc folder access
deined.
It is also work around, but it allow me to use Marco window manager which
have been officially developed for Mate desktop environment.

3. Arch Linux community is extremely active. it is even possible to find
Seamonkey for ARM64 BIT CPUS, which have been reprogrammed so Seamonkey is
supported even from Android when using Proot and Termux.
Sure, it is not newest Seamonkey, but it work.
4. I could not run Firefox or Iceweasel from Debian, I have always got
broken pipe error message when I have tried to start it.
By other words.
ArchLinux is for now my favourite distro, sure thanks to team around MR
Rausty, because every script release containspecial routines which makes
Arch Linux really usable with minimal issues.

**Comment by JanuszChmiel** [comment]
When I have typed
apt install mate-core
I have set The variable
export DEBIAN_FRONTEND=noninteractive
I have added this line as A last line in /etc/profile
I typed as Root
apt install mate-core
And there are last errors.


Processing triggers for dbus (1.12.16-1) ...

 Processing triggers for libgdk-pixbuf2.0-0:arm64 (2.38.1+dfsg-1) ...

 Errors were encountered while processing:

  udisks2

  gvfs-daemons

  gvfs-backends

  gvfs:arm64

  mate-desktop-environment-core

  caja

  mate-core

 E: Sub-process /usr/bin/dpkg returned an error code (1)

The question is who could solve this. If Debian packager or Template
creators and if it is possible to positively solve this issue from The
perspective of yours script to install Debian.
